src/Bridges/Variable/bridges/hermitian.jl

```diff
@@ -27,63 +27,48 @@
   * [`MOI.VectorOfVariables`](@ref) in [`MOI.PositiveSemidefiniteConeTriangle`](@ref)
   * [`MOI.ScalarAffineFunction{T}`](@ref) in [`MOI.EqualTo{T}`](@ref)
 
-## Duality notes
+## Reformulation
 
-Suppose for simplicity that the elements of a `2n x 2n` matrix are ordered as:
-```
-\\ 1 |\\ 2
- \\  | 3
-  \\ |4_\\
-   \\  5
-    \\
-     \\
-```
-Let `H = HermitianToSymmetricPSDBridge(n)`,
-`S = PositiveSemidefiniteConeTriangle(2n)` and `con_11_22` (resp. `con_12_21`,
-`con12diag`) be the set of `2n x 2n` symmetric matrices such that the block `1`
-and `5` are equal (resp. `2` and `4` are opposite, `3` is zero).
+The reformulation is best described by example.
 
-We consider the cone `P = S ∩ con_11_22 ∩ con_12_21 ∩ con12diag`. We have
-`P = A * H` where
-```
-    [I  0]
-    [0  I]
-A = [0  0]
-    [0 -I]
-    [I  0]
-```
-Therefore, `H* = A* * P*` where
+The Hermitian matrix:
+```math
+\\begin{bmatrix}
+  x_{11}            & x_{12} + y_{12}im & x_{13} + y_{13}im\\\\
+  x_{12} - y_{12}im & x_{22}            & x_{23} + y_{23}im\\\\
+  x_{13} - y_{13}im & x_{23} - y_{23}im & x_{33}
+\\end{bmatrix}
 ```
-     [I 0 0  0 I]
-A* = [0 I 0 -I 0]
+is positive semidefinite if and only if the symmetric matrix:
+```math
+\\begin{bmatrix}
+    x_{11} & x_{12} & x_{13} & 0       & y_{12}  & y_{13} \\\\
+           & x_{22} & x_{23} & -y_{12} & 0       & y_{23} \\\\
+           &        & x_{33} & -y_{13} & -y_{23} & 0      \\\\
+           &        &        & x_{11}  & x_{12}  & x_{13} \\\\
+           &        &        &         & x_{22}  & x_{23} \\\\
+           &        &        &         &         & x_{33}
+\\end{bmatrix}
 ```
-Moreover, as `(S ∩ T)* = S* + T*` for cones `S` and `T`, we have
-```
-P* = S* + con_11_22* + con_12_21* + con12diag*
-```
-the dual vector of `P*` is the dual vector of `S*` for which we add in the
-corresponding entries the dual of the three constraints, multiplied by the
-coefficients for the `EqualTo` constraints.
+is positive semidefinite.
 
-Note that these contributions cancel out when we multiply them by `A*`:
```

Comment:
I'd prefer keeping these duality notes. If you don't want to put them in the docstring, we can just add them as comment above `ConstraintDual`

Reply:
Yeah, they shouldn't go in the docstring. It's not really meant for consumers of the bridge, just for people wanting to grok the particular implementation of the function.